plugins/hls-code-range-plugin/test/testdata/selection-range/Import.hs

```diff
@@ -1,4 +1,4 @@
 module MultiPositions where
 
 import Data.List (find)
-import qualified Data.Foldable (foldl, foldl')
+import qualified Data.Foldable (foldl, foldl')
```

Comment:
Modified this file by accident?

Reply:
Ah yes, stylish-haskell was adding white spaces in the import lines, and it was messing up the selection ranges defined in the tests, I'll revert it.

Reply:
Reverted it back to normal

Reply:
Oh, maybe we could run the formatter in a separate PR.

Reply:
Yep, We'll also have to edit the test case file.